<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaipanFrame</class>
 <widget class="QFrame" name="PaipanFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>4</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radio_time1">
        <property name="text">
         <string>手表时</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_time2">
        <property name="text">
         <string>平太阳时</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_time3">
        <property name="text">
         <string>真太阳时</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radio_modeb">
        <property name="text">
         <string>八字</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_modeh">
        <property name="text">
         <string>河洛</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_modez">
        <property name="text">
         <string>紫微</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>PaipanFrame</receiver>
   <slot>onSelPerson(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>PaipanFrame</receiver>
   <slot>onDbclkPerson(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>PaipanFrame</receiver>
   <slot>onCloseTab(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSelPerson(QListWidgetItem*)</slot>
  <slot>onDbclkPerson(QListWidgetItem*)</slot>
  <slot>onCloseTab(int)</slot>
 </slots>
</ui>
